### PR TITLE
Create Category.java

### PR DESCRIPTION
**Pull request by Alicewillbe**

### CODE REVIEW

File: src/main/java/com/spare4fun/core/entity/Category.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package flag_camp_2020_t3.com.spare4fun.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+
```
Comment (Alicewillbe) [inline src/main/java/com/spare4fun/core/entity/Category.java:7]: You should give java doc for this class. Please refer to doc https://google.github.io/styleguide/javaguide.html
Or please refer to branch entity-user

**Review by Alicewillbe — COMMENTED** [review]
@joannadengdeng

File: src/main/java/com/spare4fun/core/entity/Category.java (hunk 1 above)
Comment (Alicewillbe) [reply src/main/java/com/spare4fun/core/entity/Category.java:15]: You should never upload Secret.java, please change it.

**Comment by Alicewillbe** [comment]
You should never upload Secret.java, please change.

File: src/main/java/com/spare4fun/core/entity/Category.java (hunk 1 above)
Comment (joannadengdeng) [reply src/main/java/com/spare4fun/core/entity/Category.java:15]: I have added the java doc in the Category class.

**Comment by joannadengdeng** [comment]
> You should never upload Secret.java, please change.

I have already deleted Secret.java

**Pull request merged**